source/extensions/filters/common/ext_authz/ext_authz_http_impl.cc

```diff
@@ -343,21 +343,19 @@ ResponsePtr RawHttpClientImpl::toResponse(Http::ResponseMessagePtr message) {
   // headers_to_remove in a variable first.
   std::vector<Http::LowerCaseString> headers_to_remove;
   if (status_code == enumToInt(Http::Code::OK)) {
-    // iterate() over the headers rather than just get() in case the storage
-    // header occurs more than once.
-    absl::string_view storage_header_name_view = absl::string_view(storage_header_name.get());
-    message->headers().iterate([&](const Http::HeaderEntry& entry) {
-      if (entry.key() == storage_header_name_view) {
-        absl::string_view storage_header_value = entry.value().getStringView();
+    const auto& get_result = message->headers().getAll(storage_header_name);
+    for (size_t i = 0; i < get_result.size(); ++i) {
```

Comment:
Can we do "modern" loop here?

Reply:
It's not supported, because this type `GetResult` does not implement `begin`/`end`:
https://www.cprogramming.com/c++11/c++11-ranged-for-loop.html

Reply:
👍